saleor/graphql/checkout/mutations.py

```diff
@@ -36,8 +37,10 @@
 from ...core.tracing import traced_atomic_transaction
 from ...core.transactions import transaction_with_commit_on_errors
 from ...order import models as order_models
+from ...plugins.webhook.utils import from_shipping_app_id
```

Comment:
We shouldn't have any depend relation between Saleor and Plugin. Saleor should not use any plugin's logic.